tests/cypress.config.ts

```diff
@@ -0,0 +1,31 @@
+import { defineConfig } from 'cypress'
+
+export default defineConfig({
+  e2e: {
+    // Allows to run all tests in open mode
+    experimentalRunAllSpecs: true,
+    setupNodeEvents(on, config) {
+      config.chromeWebSecurity = false
+      config.video = false
+      config.baseUrl = config.env.BASEURL
+      
+      config.env = {
+        devices: {
+          desktop: ['all', 'desktop'].includes(config.env.DEVICE),
+          mobile: ['all', 'mobile'].includes(config.env.DEVICE),
+        },
+        resolution: {
+          desktop: { viewportWidth: 1440, viewportHeight: 1080 },
```

Comment:
Where do these numbers come from?

Reply:
For the width I used the value that is set in the dev tools under Laptop L. I left the height at 1080px, through experience. It is better to use smaller values than larger ones.